test/gtfs_test.exs

```diff
@@ -68,11 +128,20 @@ defmodule GtfsTest do
     test "merges stops from both directions, flipping direction 0" do
       pid =
         Gtfs.start_mocked(%{
+          "routes.txt" => [
+            "route_id,route_type",
+            "route,3"
```

Comment:
Potential trip hazard: If you forget to specify the `route_type` column in a test, (reasonably likely, since we don't include it in the final `Route.t()` struct), the route will be silently dropped. I can imagine losing a couple hours debugging something like that.

Should we make `bus_route?` crash if it's missing this column to make the problem easier to spot, or accept all routes as bus routes if the column, or something like that?

Reply:
Added a check.